docker/base-images.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+images:
+- gcr.io/fuzzbench/base-image:test-experiment
+- gcr.io/fuzzbench/base-image
+steps:
+- args:
+  - build
+  - --tag
+  - gcr.io/fuzzbench/base-image
+  - --tag
+  - gcr.io/fuzzbench/base-image:test-experiment
```

Comment:
do we need this test-experiment tag? consider removing it if not

Reply:
That tag is the default value for experiment tag (following previous review comments regarding generating this file). I'm not sure if its not needed.